expr/functions/scale/function.go

```diff
@@ -21,7 +21,7 @@ func GetOrder() interfaces.Order {
 func New(configFile string) []interfaces.FunctionMetadata {
 	res := make([]interfaces.FunctionMetadata, 0)
 	f := &scale{}
-	functions := []string{"scale"}
+	functions := []string{"scale", "scaleAfterTimestamp"}
```

Comment:
It will return the same Description for both names, richt?

Reply:
AFAIR I need to provide description for both metrics... Added one.